ricecooker/classes/nodes.py

```diff
@@ -798,3 +798,56 @@ def truncate_fields(self):
             q.truncate_fields()
 
         super(ExerciseNode, self).truncate_fields()
+
+class SlideshowNode(ContentNode):
+    """ Model representing Slideshows
+
+        Slideshows are sequences of "Slides", a combination of an image and caption.
+        Slides are shown in a specified sequential order.
+
+                Attributes:
+            source_id (str): content's original id
+            title (str): content's title
+            license (str or <License>): content's license
+            author (str): who created the content (optional)
+            aggregator (str): website or org hosting the content collection but not necessarily the creator or copyright holder (optional)
+            provider (str): organization that commissioned or is distributing the content (optional)
+            description (str): description of content (optional)
+            files ([<SlideImageFile>]): images associated with slides
+            thumbnail (str): local path or url to thumbnail image (optional)
+            extra_fields (dict): any additional data needed for node (optional)
+            domain_ns (str): who is providing the content (e.g. learningequality.org) (optional)
+
+    """
+    kind = content_kinds.SLIDESHOW
+
+    def __init__(self, source_id, title, slideshow_data=None, **kwargs):
+        if not slideshow_data:
+            slideshow_data = []
+
+        for idx, file in enumerate(kwargs['files']):
+            filename = file.get_filename()
+
+            checksum, ext = ['', '']
+
+            if filename:
+                checksum, ext = filename.split('.')
+
+            slideshow_data.append(
+                {
+                    'caption': file.caption,
+                    'sort_order': idx,
+                    'checksum': checksum,
+                    'extension': ext
+                }
+            )
+
+        super(SlideshowNode, self).__init__(source_id, title, extra_fields=slideshow_data, **kwargs)
```

Comment:
1. need to pass along `license` after `title` to base class' `__init__`  

2. also note extra_fields should be a dictionary, not a list (see https://github.com/learningequality/ricecooker/blob/master/ricecooker/classes/nodes.py#L395)